english/earlyBird/content.md

```diff
@@ -0,0 +1,13 @@
+Advantages early birds have over night owls
+-------------------------------------------
+
+1. Becoming an early riser will make you more successful.
+2. Morning people are happier than night owls.
+3. Morning people are often in better shape than night owls.
+4. Waking up early leads to increased productivity.
+5. Possess healthy character traits.
+
+So kudos to you for being an early bird.
+
+Click [here](../marshmallow.md) to go back to the story.
+Click [here](https://www.google.com/) to go to google.
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.